### PR TITLE
feat(CI): Expand copyright checks

### PR DESCRIPTION
**Pull request by tibetiroka**
**CI/CD/Testing**

This PR adds a new check for the copyright file that will actually detect issues.

I also removed an extra empty line from before the check.

## Usage examples
#9853

You can see the output of the check [here](https://github.com/endless-sky/endless-sky/actions/runs/8124308308/job/22205636631).

## Testing Done
#9853 and testing on various other control files, unit tests etc. The action was also tested, but it's also tested in this PR.

### CODE REVIEW

**Review by OcelotWalrus — APPROVED** [review]
Seems good for me.

File: .github/workflows/projects_check.yml [hunk 1]
```diff
@@ -55,8 +54,15 @@ jobs:
         show-progress: false
     - name: Install dependencies
       run: python3 -m pip install --user python-debian
-    - name: Validate copyright file
+    - name: Validate copyright file with python-debian
       run: python3 ./utils/check_copyright.py
+    - name: Validate copyright file with debian-control-linter
+      uses: tibetiroka/debian-control-linter-action@v1
```
Comment (quyykk) [inline .github/workflows/projects_check.yml:60]: Is your action not doing a superset of the check above?

**Review by quyykk — COMMENTED** [review]
Peter mentioned in the past that he's not a fan of using our own (personal) actions for ES. I don't really mind IMO, but let's hear what others think.

File: .github/workflows/projects_check.yml (hunk 1 above)
Comment (tibetiroka) [reply .github/workflows/projects_check.yml:60]: I think so, but since these checks are really fast I don't see an issue with having them both.

**Comment by warp-core** [comment]
Can this be added to the high DPI repo as well?

**Comment by tibetiroka** [comment]
Yes, I just wanted to get it into ES first to avoid editing the config in two places :)

File: .github/workflows/projects_check.yml (hunk 1 above)
Comment (quyykk) [reply .github/workflows/projects_check.yml:60]: I guess it would be one less script to maintain. @warp-core thoughts?

Comment (warp-core) [reply .github/workflows/projects_check.yml:60]: I think it's fine to keep it for now.

**Pull request merged**